### PR TITLE
Add right-to-left and top-to-bottom tests for text-wrap: balance

### PR DESCRIPTION
**Pull request by aikocny**
Inspired from text-wrap-balance-001.html, I changed the array elements to be Arabic (right-to-left) and Chinese (top-to-bottom, right-to-left) characters to support testing for non-English characters.

### CODE REVIEW

**Comment by nt1m** [comment]
Harness status seems to be error for both of the tests in all browsers (see the checks tab)

**Comment by aikocny** [comment]
Fixed it, there was an error with duplicate subtest name.

File: css/css-text/white-space/text-wrap-balance-top-to-bottom.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  ]) {
+  const normal = document.createElement('div');
+  const balance = document.createElement('div');
+  normal.textContent = text;
+  balance.textContent = text;
+  balance.classList.add('balance');
+  container.appendChild(normal);
+  container.appendChild(balance);
+  test(() => {
+    assert_equals(normal.offsetHeight, balance.offsetHeight);
```
Comment (nt1m) [inline css/css-text/white-space/text-wrap-balance-top-to-bottom.html:32]: I think you want to compare the `offsetWidth` here given in vertical writing mode height and width are reversed.

File: css/css-text/white-space/text-wrap-balance-top-to-bottom.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+.balance {
+  text-wrap: balance;
+}
+</style>
+<div id="container"></div>
+<script>
+const container = document.getElementById('container');
+for (const text of [
+    '平衡应该',
```
Comment (nt1m) [inline css/css-text/white-space/text-wrap-balance-top-to-bottom.html:20]: This seems to fit in a single line within height: 15ch, so balancing won't have an effect on this.

**Review by nt1m — APPROVED** [review]

File: css/css-text/white-space/text-wrap-balance-top-to-bottom.html (hunk 2 above)
Comment (aikocny) [reply css/css-text/white-space/text-wrap-balance-top-to-bottom.html:20]: Yes it is the intention to test strings that displays as one, two, and three lines. To make sure that balancing will not add the number of lines.

File: css/css-text/white-space/text-wrap-balance-top-to-bottom.html (hunk 1 above)
Comment (aikocny) [reply css/css-text/white-space/text-wrap-balance-top-to-bottom.html]: Good catch, just committed a change.

**Pull request merged**